deploy/cw-bundle.yaml

```diff
@@ -9,9 +9,9 @@ metadata:
   annotations:
```

Comment:
@egegunes can we remove the namespace feald for CW under this PR as we discussed yesterday

Reply:
https://github.com/percona/percona-postgresql-operator/pull/450/commits/dea752c001f0eaa1975407db476d038be4b2ef95